Draft release notes for Klaw documentation (#42)
aiven/klaw-docs: docs/releases/release210.rst

- **Approve & Reject Schema Requests**: Users can now approve or reject schema requests using the new UI.

.. image:: /../../../_static/images/topic/ApprovalRequestTopic-react.png

To preview the new Klaw user user interface, open the ``application.properties`` file on the Klaw **core** module, and set the value of the following property to ``true``:
::
    # Enable new Klaw user interface
    klaw.coral.enabled=true

.. note::
    We are taking an incremental, feedback-driven approach in rolling out the new Klaw interfaces. By providing the feature flag to preview user interfaces, we would like you to share your valuable `feedback <https://github.com/aiven/klaw/issues/new?assignees=&labels=&template=03_feature.md>`_.

Confluent Cloud Kafka
Klaw now supports Confluent Cloud kafka Topics, Acls.
Apis implemented :
https://docs.confluent.io/cloud/current/api.html#tag/Topic-(v3)
https://docs.confluent.io/cloud/current/api.html#tag/ACL-(v3)

For more information, see `Confluent Cloud <https://www.klaw-project.io/docs/howto/clusterconnectivity/confluent-cloud-kafka-cluster-ssl-protocol>`_.

> Switching Teams

Reviewer: ```suggestion
Switching Teams
~~~~~~~~~~~~
```

Reviewer 2: Suggest calling it "Switch Teams"

Release 2.1.0
=============

Date: 3rd of March, 2023

Overview
--------

Klaw version 2.1.0 is a minor release with several new features and improvements. Key highlights of this release include redesigned approval and decline screens in the React UI, Confluent Cloud support, The ability to switch between teams, codebase optimizations, and additional enhancements.

What's new in Klaw 2.1.0
------------------------

Redesigned Request screens with React UI
`````````````````````````````````````````
With the switch to React, this release features a redesign of the following user interfaces:

- **Approve & Reject Topic Requests**:  Users can now approve or reject topic requests using the new UI.

- **Approve & Reject Subscription Request** (Producer/Consumer): Users can now approve or reject ACL requests for both producers and consumers using the new UI.

> - **Approve & Reject Schema Request**: Users can now approve or reject schema requests using the new UI.

Reviewer: ```suggestion
- **Approve & Reject Schema Requests**: Users can now approve or reject schema requests using the new UI.
```

.. image:: /../../../_static/images/topic/ApprovalRequestTopic-react.png

> 

Author: This image is missing and is needed.

> To preview the new Klaw user user interface, open the ``application.properties`` file on the Klaw **core** module, and set the value of the following property to ``true``:

Reviewer 2: ```suggestion
To preview the new Klaw user interface, open the ``application.properties`` file on the Klaw **core** module, and set the value of the following property to ``true``:
```

::
    # Enable new Klaw user interface
    klaw.coral.enabled=true

.. note::
    We are taking an incremental, feedback-driven approach in rolling out the new Klaw interfaces. By providing the feature flag to preview user interfaces, we would like you to share your valuable `feedback <https://github.com/aiven/klaw/issues/new?assignees=&labels=&template=03_feature.md>`_.

> Confluent Cloud

Reviewer: ```suggestion
Confluent Cloud Kafka
```

> `````````````````

Reviewer: ```suggestion
``````````````
```

> Klaw now supports Confluent Cloud Topics, Acls.

Reviewer: ```suggestion
Klaw now supports Confluent Cloud Kafka Topics, Acls.
```

> Klaw now supports Confluent Cloud kafka Topics, Acls.
> Apis implemented :
> https://docs.confluent.io/cloud/current/api.html#tag/Topic-(v3)
> https://docs.confluent.io/cloud/current/api.html#tag/ACL-(v3)

Reviewer 2: ```suggestion
Klaw now offers Confluent Cloud Kafka Topics and ACLs support, expanding its capabilities to interact with Confluent Cloud Kafka. This release includes the implementation of the following APIs:
https://docs.confluent.io/cloud/current/api.html#tag/Topic-(v3)
https://docs.confluent.io/cloud/current/api.html#tag/ACL-(v3)
```

For more information, see `Confluent Cloud <https://www.klaw-project.io/docs/howto/clusterconnectivity/confluent-cloud-kafka-cluster-ssl-protocol>`_.

Switching Teams

> `````````````````````````````````````````````````````

Reviewer: ```suggestion
```````````````
```

> With this release, users now have the ability to be a member of multiple teams and switch between those teams without logging out and logging back in. Allowing a single user to have proper auditable capabilities while also working across multiple teams.

Reviewer 2: ```suggestion
Klaw now allows users to be members of multiple teams and easily switch between them without needing to log out and back in. With this new functionality, a single user can now have proper auditable capabilities while working across multiple teams. Additionally, users will benefit from increased flexibility and efficiency when working within the Klaw platform, particularly when creating or approving requests.
```
